newIDE/app/src/ObjectEditor/Editors/SpriteEditor/AnimationPreview.js

```diff
@@ -37,9 +39,11 @@ type Props = {|
   initialZoom?: number,
   fixedHeight?: number,
   fixedWidth?: number,
+  isPrivate?: boolean,
```

Comment:
This is named too generically. What is private: the object, the sprite, the image, the project?